.flake8

```
@@ -6,7 +6,7 @@ ban-relative-imports = True
 format-greedy = 1
 inline-quotes = double
 enable-extensions = TC, TC1
-type-checking-exempt-modules = typing, typing-extensions
+strict-type-checking = true
```

```suggestion
type-checking-strict = true
```
https://github.com/snok/flake8-type-checking#strict ?